tracer/src/Datadog.Trace/AppSec/Waf/Context.cs

```diff
@@ -42,6 +42,7 @@ public IResult Run(ulong timeoutMicroSeconds)
                 ThrowHelper.ThrowException("Can't run WAF when disposed");
             }
 
+            _stopwatch.Start();
```

Comment:
I assume you want to restart the timer from 0, not continue from where you left off?
```suggestion
            _stopwatch.Restart();
```